### DISCUSSION TITLE
How to skip config.fish lines in file preview terminal (bat)

### DISCUSSION THREAD

**Discussion by andeledea** [PatrickF1/fzf.fish#254] [Q&A] [answered]
In my config.fish I added some fish startup commands (neofetch, fish_greeting...).
When I use ctrl+alt+F to search in the directory, in the side terminal that shows the file preview all these commands are executed so I end up with neofetch on top of each preview and I have to scroll down to see the output of bat (figure shows end of neofetch and start of file).

![fzfneof](https://user-images.githubusercontent.com/62064962/182942283-4076b0ea-53d9-47b1-a700-19db0df2b7f9.png)

Is there a way to open the side terminal without all the "init" commands (or only some of them)?
Thank you

**Comment by PatrickF1** [comment, marked as answer]
In config.fish, put all your interactive changes that print stuff under
`status is-interactive || exit`

Explanation for why this works and is necessary can be found in Related: https://github.com/PatrickF1/fzf.fish/discussions/182.
